pom.xml

```diff
@@ -57,8 +55,25 @@
         <version.arm-oraclelinux-wls-cluster-deletenode>1.0.1</version.arm-oraclelinux-wls-cluster-deletenode>
         <!--  versions  end -->
 
+        <git.tag>main</git.tag>
+        <git.repo>weblogic-azure</git.repo>
+        <git.repo.owner>oracle</git.repo.owner>
+        <artifactsLocationBase>https://raw.githubusercontent.com/${git.repo.owner}/${git.repo}/${git.tag}/weblogic-azure-vm</artifactsLocationBase>
+
+        <module.basedir>${project.basedir}</module.basedir>
+        <template.validation.tests.directory>${module.basedir}/../arm-ttk/arm-ttk</template.validation.tests.directory>
+
+        <template.azure-common.properties.url>file:///${module.basedir}/resources/azure-common.properties</template.azure-common.properties.url>
```

Comment:
I don't know what this one does.  I don't see any usages of this property in any file:

```bash
% find . -type f -exec grep template.azure-common.properties.url {} \; -print
find . -type f -exec grep template.azure-common.properties.url {} \; -print
        <template.azure-common.properties.url>file:///${module.basedir}/resources/azure-common.properties</template.azure-common.properties.url>
./pom.xml
%
```

Can this be removed?

Reply:
@edburns 
`template.azure-common.properties.url`, this is used in project [azure-javaee-iaas ](https://github.com/azure-javaee/azure-javaee-iaas), see [here](https://github.com/azure-javaee/azure-javaee-iaas/blob/b7b966b502212c40f23fd391a088da6a9b20bdc3/arm-parent/pom.xml#L361)

`template.validation.tests.directory`, [it](https://github.com/azure-javaee/azure-javaee-iaas/blob/b7b966b502212c40f23fd391a088da6a9b20bdc3/README.md?plain=1#L34) is used to define the path of arm-ttk.

`module.basedir` is a value I extract to refactor the code .